dirty_cat/test/test_minhash_encoder.py

```diff
@@ -56,25 +58,70 @@ def test_get_unique_ngrams():
     enc = MinHashEncoder(n_components=2)
     ngrams = enc.get_unique_ngrams(string, ngram_range)
     assert ngrams == true_ngrams
-        
+
+
 def profile_encoder(Encoder, hashing='fast', minmax_hash=False):
     # not an unit test
-    
+
     from dirty_cat import datasets
     import pandas as pd
     employee_salaries = datasets.fetch_employee_salaries()
-    data = pd.read_csv(employee_salaries['path'])
-    X = data['Employee Position Title'].tolist()
+    data = employee_salaries['data']
+    X = data['employee_position_title'].tolist()
     X = X * 10
     t0 = time.time()
-    encoder = Encoder(n_components=50, hashing=hashing, minmax_hash=minmax_hash)
+    encoder = Encoder(n_components=50, hashing=hashing,
+                      minmax_hash=minmax_hash)
     encoder.fit(X)
     y = encoder.transform(X)
     assert y.shape == (len(X), 50)
     eta = time.time() - t0
     return eta
 
 
+@pytest.mark.parametrize("input_type", ['list', 'numpy', 'pandas'])
+@pytest.mark.parametrize("missing", ['aaa', 'error', ''])
+@pytest.mark.parametrize("hashing", ["fast", "murmur"])
```

Comment:
Sorry to nitpick, but the tensor-product of options here leads to a expensive combinatorial: this test is taking more time than it should. Could would downsample here? Some think like:
```
@pytest.mark.parametrize("input_type,missing,hashing", [
                                               ['list', 'aaa', 'fast'],
                                               ...
                                           ])
```
for a few well-chosen values